app/ChordNode.ts

```diff
@@ -1047,8 +1047,12 @@ export class ChordNode {
       console.error(`stabilize: updateSuccessorTable failed with `, err);
     }
 
-    console.log(`--\n{${this.id}}.predecessor = ${this.predecessor.id}`);
```

Comment:
You couldn't resist removing my console tells!! :-)  Do they really bother you that much?  I'll have to figure out why :-)

Reply:
It's mostly because it gets too noisy when I run like 25 or more nodes using docker-compose. All the nodes print to a single console, and the errors fly off the screen really quickly.